src/index.js

```diff
@@ -7,6 +7,7 @@ function getDefaults() {
     memoizeFallback: false,
     bindI18n: '',
     bindI18nStore: '',
+    parseErrorHandler: (err, options) => { return options.defaultValue }
```

Comment:
I believe it's possible for a default value not to be given, in which case this should probably return the key

Reply:
This should also probably try to parse with the default value and then if that fails return the key

Reply:
@jesseyay I will accept as is or with an improved default handling like suggested by @keattang

Reply:
Awesome!
I like the extra flexibility of having the key so I have added it in as a parameter for `parseErrorHandler`